libs/design/src/atoms/form/checkbox/checkbox.md

```diff
@@ -0,0 +1,78 @@
+# DaffCheckbox and DaffCheckboxSet
+The `DaffCheckboxComponent` is used to toggle an individual selection in a form. It can be hooked into Angular's `FormControl` to accomodate custom functionality. The `DaffCheckboxSetComponent` serves as a wrapper around a logical group of radios to provide styling.
+
+## Accessibility
+
+The `DaffCheckboxComponent` uses native `<input>` HTML elements to ensure an accesible experience by default.
+
+## DaffCheckbox Inputs
```

Comment:
Can we combined `DaffCheckbox Inputs` and `DaffCheckbox Outputs` into one section called `DaffCheckbox Properties`.

Similarly, `DaffCheckboxSet Inputs` and `DaffCheckboxSet Functions` into one section called `DaffCheckboxSet Properties`?

Or are we purposely trying to create separation between inputs, outputs, and other functions?

Reply:
I like this idea a lot, no intent to separate them.  I think we could add just an extra column to the table called something like `property` or `property type` and fill it in with `input`, `output`, and `function`.

Reply:
updated in 73d1c0fef1a29a953f3ee8fc2da0b0801e61cc0c